README.md

```diff
@@ -123,6 +124,11 @@ Submit issues at: https://github.com/ocp-power-automation/openshift-install-powe
 
 ```
 
+## Usage with Containers
```

Comment:
This should be a sub section within the "Usage"

Reply:
IMHO the main README should just have information on how to run it under the "Usage" section. The process to build the container image should be a link to the another doc